CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Version 2.0.0
+
+-   Feat: implement CMB2 and replace Metabox.io
```

Comment:
Formatting hier klopt niet helemaal met de rest van de file, heading vs no heading. Ik heb geen voorkeur maar just saying.

Reply:
Is aangepast, scherp ;)